src/main/java/org/traccar/protocol/NavtelecomProtocolDecoder.java

```diff
@@ -193,8 +193,11 @@ protected Object decode(
                     Position position = new Position(getProtocolName());
                     position.setDeviceId(deviceSession.getDeviceId());
 
-                    for (int j = 0; j < bits.length(); j++) {
+                    for (int j = 0, l = 2; j < bits.length(); j++, l++) {
```

Comment:
Why do we need a second variable? Can't we just add 2 to `j`?

Reply:
Yes, of course, we can do this:

> In order for everything to be calculated correctly, we need to do something like this `(j + 2 - 45)`.

I just compared both options and it seemed to me that with one more variable, the option looks easier.
And in the rest of the protocol, there are still many parameters that need to be added in this way.

I can remove the variable and replace `l` with `j + 2`.